File: tests/rspec/pages/utilities/stats.rb
class Stats < ControlPanelPage
	set_url_matcher /utilities\/stats/

	element :heading, 'div.box form h1'

	element :alert, 'div.alert'

	element :content_table, 'div.box form table'
	elements :rows, 'div.box form table tr'
	elements :sources, 'div.box form table tr td:first-child'
	elements :counts, 'div.box form table tr td:nth-child(2)'

	element :bulk_action, 'form fieldset.tbl-bulk-act select[name="bulk_action"]'
	element :action_submit_button, 'form fieldset.tbl-bulk-act input.submit'

	def load
		self.open_dev_menu
		click_link 'Utilities'
		click_link 'Statistics'
	end
end

File: tests/cypress/cypress/elements/pages/utilities/Stats.js
import ControlPanel from '../ControlPanel'

class Stats extends ControlPanel {
    constructor() {
        super()
        this.url = 'admin.php?/cp/utilities/stats';

        this.selectors = Object.assign(this.selectors, {
        	  "heading": '.ee-main .title-bar .title-bar__title',
			  "content_table": 'div.container form table',
			  "rows": 'div.container form table tr',
			  "sources": 'div.container form table tr td:first-child',
			  "counts": 'div.container form table tr td:nth-child(2)'
        })
    }

    load(){
	    this.open_dev_menu()
	    cy.get('body > .dropdown').find('a:contains("Utilities")').click()
	    this.get('wrap').find('a:contains("Statistics")').click()
    }


}
export default Stats;